agera/src/main/java/com/google/android/agera/BaseObservable.java

```diff
@@ -38,149 +39,131 @@
  */
 public abstract class BaseObservable implements Observable {
   @NonNull
-  private final Worker worker;
+  private static final Object[] NO_UPDATABLES_OR_HANDLERS = new Object[0];
+  @NonNull
+  private final WorkerHandler handler;
+  private final int shortestUpdateWindowMillis;
+
+  @NonNull
+  private Object[] updatablesAndHandlers;
+  private int size;
+  private long lastUpdateTimestamp;
```

Comment:
we're grouping by final/non-final in the rest of the codebase though, so not really possible to group by use case as such (and follow the current style)